src/coord/radar/Radar.ts

```diff
@@ -67,6 +67,7 @@ class Radar implements CoordinateSystem, CoordinateSystemMaster {
             indicatorAxis.name = indicatorModel.get('name');
             // Inject model and axis
             indicatorAxis.model = indicatorModel;
+            indicatorAxis.inverse = indicatorModel.get('inverse');
```

Comment:
This is probably not necessary to be set to `indicatorAxis.inverse`. You should call `indicatorModel.get('inverse')` whenever you need the value.

Reply:
Thanks for the information. Yes, others may depend on this information.